src/app/views/sidebar/sample-queries/SampleQueries.tsx

```diff
@@ -448,7 +467,7 @@ export class SampleQueries extends Component<ISampleQueriesProps, any> {
           items={sampleQueries}
           selectionMode={SelectionMode.none}
           columns={columns}
-          groups={groups}
+          groups={this.groups}
```

Comment:
why was `this` added?

Reply:
I saved it as a variable instead cause it was regenerating the groups